src/teleop/gui/src/components/MoteusStateTable.vue

```diff
@@ -1,101 +1,189 @@
 <template>
   <div class="wrap">
-    <div>
-      <h3>Moteus States</h3>
+    <div v-if="motor">
```

Comment:
This feels like a bit of a strange way of doing things. and makes this component pretty large, I would either add comment explaining what each of these v-if cases are or better yet just make the arm moteus state table a separate component from the drive one.

Reply:
Ya, I also think allowing for flexibility would be nice (make the table adjust based on the msg size rather than hard coding the indices)